omnibus/.kitchen.yml

```diff
@@ -93,7 +94,9 @@ platforms:
       # gets cranky if the mounted Chef source folder is a symlink. This
       # mounts at `C:\vagrant\code` and the Chef source folder is available
       # at `C:\vagrant\code\chef`
-      - ['../..', '/vagrant/code']
+      - ['..', '/vagrant/code/chef']
```

Comment:
@mwrock is this feedback resolved? I saw https://github.com/chef/chef/pull/5042#issuecomment-227488991, but this note didn't go away, so I'm feeling like this line wasn't changed after this feedback.